src/GoodsList.tsx

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
+import ListGroup from 'react-bootstrap/ListGroup';
 import { Good } from './types/Good';
 
 type Props = {
-  goods: Good[]
+  goods: Good[];
 };
 
-export const GoodsList: React.FC<Props> = ({ goods }) => (
-  <ul>
-    {goods.map(good => (
-      <li key={good.id} data-cy="good">
-        {good.name}
-      </li>
-    ))}
-  </ul>
+export const GoodsList: React.FC<Props> = React.memo(
+  ({ goods }) => (
+    <ListGroup>
+      {goods.map(good => {
+        const { id, color, name } = good;
+
```

Comment:
When you don't need to pass the whole object further (in this case variable `good`) then it's better to write like this:
```suggestion
      {goods.map(({ id, color, name }) => {
```
It's one line shorter and shows that we use only specific field and don't need the whole object